### PR TITLE
Ft/update readme file

### PR DESCRIPTION
**Pull request by AdelineA**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
-## Learn More
+## Getting Started
+To get started with the Co-Design project, you can follow the steps below to clone the repository, install dependencies, and run the development server.
 
-To learn more about Next.js, take a look at the following resources:
+## Prerequisites
+Before you begin, ensure you have the following installed on your machine:
+1. Node.js (version >= 14)
+2. npm
+3. git
```
Comment (kareraolivier) [inline README.md:21]: Also, add how you can install them

File: README.md [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
-## Deploy on Vercel
+```bash
+npm run dev
+```
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+Open your browser and navigate to [http://localhost:3000](http://localhost:3000) to see the application running.
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+## Deployment
```
Comment (kareraolivier) [inline README.md:40]: Here It can be Deployed-link or Link

File: README.md [hunk 3]
```diff
@@ -1,40 +1,41 @@
...
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+## Table of Content
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+- [Introduction](#introduction)
+- [Get Started](#getting-started)
+- [Prerequisites](#prerequisites)
+- [Deployment](#deployment)
```
Comment (kareraolivier) [inline README.md:8]: Here before get started you can also add a title for the technology used like:

Screenshot Capture - 2024-06-18 - 10-19-59

**Review by kareraolivier — COMMENTED** [review]
you can use this readme as an example:

https://github.com/othneildrew/Best-README-Template

File: README.md (hunk 1 above)
Comment (AdelineA) [reply README.md:21]: fixed

File: README.md (hunk 2 above)
Comment (AdelineA) [reply README.md:40]: Fixed

File: README.md (hunk 3 above)
Comment (AdelineA) [reply README.md:8]: Fixed

**Comment by dmohns** [comment]
Hey @AdelineA and @kareraolivier 

What about this PR? Is this good to go?

**Comment by AdelineA** [comment]
Hello @dmohns ,
This pr is good to go.

**Comment by dmohns** [comment]
Thank you 🙏

**Pull request merged**